### ISSUE TITLE
HUD UI Elements

### ISSUE THREAD

**Issue by FiestaMasta** [flatworld-studios/joe-liebermans-blood-urge#10]
Health, Stamina, Ammo

**Comment by coleswilson** [comment]
I've done a few tutorial on the HUD. I think I can take a swing at this and have it ready to plug in the variables set from the "Character Statistics"

**Comment by coleswilson** [comment]
@FiestaMasta what is the difference between this and issues #7?

**Comment by FiestaMasta** [comment]
Issue #7 is a "Epic". If you look at it from GitHub is doesn't make a lot of sense. But Epics are used to organize the various issues. So This issue #10 is technically a subset of #7.

**Comment by FiestaMasta** [comment]
We might want to think about breaking this down. As creating a Health Bar, Ammo Bar, Stamina Bar and Game Score at the top of the screen is going to consist of very different processes.

**Comment by coleswilson** [comment]
The processes only vary slightly from my understanding of the tutorials that I've done. They just pull the data from variables established in other parts of the game, into a HUD widget, and write it to the HUD in the manner of your choosing. I'll take a swing, if it get's vastly different I'll break it down.

**Comment by FiestaMasta** [comment]
You are right, the process for displaying the information will be very similar. However the generation of the data you are going to display is very different is what I meant.

Health, Stamina, Ammo, Flag Captures. All these data points will likely be finished at very different times.

**Comment by FiestaMasta** [comment]
Either way, no need to needlessly complicate things. This is just for organization so it's up to you, especially if you are taking it on.
